### ISSUE TITLE
expand_wildcards:closed does not work to indices with alias

### ISSUE THREAD

**Issue by childe** [elastic/elasticsearch#10591]
If the index logstash-2015.04.14 has an alias named logstash-2015.04.14-test, 
I could get result from logstash-*/_settings?expand_wildcards=closed.

I am not sure if it is bug ?

**Comment by clintongormley** [comment]
So the wildcard is matching the alias, which points to the closed index.  Interesting...

**Comment by javanna** [comment]
We used not to check the state of concrete indices after aliases resolution, which might explain this behaviour. That should be fixed with #9057 though. @childe which version of elasticsearch are you running on?

**Comment by childe** [comment]
1.4.4
I thought logstash-*/_settings?expand_wildcards=closed would NOT get result of open indices. am I right?
logstash-2015.04.14  is open , but i got it from the api.

**Comment by javanna** [comment]
I can confirm this is a bug, could repro with 1.5.1 too.

```
curl -XPUT localhost:9200/test
curl -XPUT localhost:9200/test/_alias/testaliasopen
curl -XPUT localhost:9200/testclosed
curl -XPUT localhost:9200/testclosed/_alias/testaliasclosed

#following call returns closed index too
curl 'localhost:9200/test*/_settings?pretty&expand_wildcards=open'

#following call returns open index too
curl 'localhost:9200/test*/_settings?pretty&expand_wildcards=closed'
```

**Comment by clintongormley** [comment]
This has been fixed in 2.2.0
